src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -16,13 +16,69 @@
     <cd-card title="Status"
              i18n-title
              class="col-sm-6 px-3">
-             Text
+      <div class="d-flex ml-4 mr-4 mb-5"
+           *ngIf="healthData$ | async as healthData">
+        <i *ngIf="healthData.health?.status"
+           [ngClass]="[healthData.health.status | healthIcon, icons.large]"
+           [ngStyle]="healthData.health.status | healthColor"
+           [title]="healthData.health.status"></i>
+        <span class="ml-2 mt-n1"
+              i18n>Cluster</span>
+      </div>
+      <section class="border-top mt-5"
+               *ngIf="isAlertmanagerConfigured && (crticialActiveAlerts || warningActiveAlerts)">
+        <div class="d-flex flex-wrap ml-4 mr-4">
+          <span class="pt-2"
+                i18n>Alerts</span>
+          <!-- Potentially make widget component -->
+          <button class="btn btn-outline-danger rounded-pill ml-2"
```

Comment:
In earlier dashboard we have tool tip for dashboard section to overview for users,
I think that is missing in new dashboard main page over here,
can we add tool tip for all main section, even users may not able to understand red yellow alerts it's good to provide tool tip for those as well,

Reply:
Yup, right now there is a tooltip (title) in the button which will say if its a `Danger` alert or `Warning` alert. But the current tooltip which exists on the dashboard should be reworked. And it can only be done after we deliver the entire stuffs. Thanks for pointing out!